### PR TITLE
Update and improvements

### LINKED ISSUE #11: Add a way to quickly minimize bar
Can be done with a close or X button in the top right

### LINKED ISSUE #12: Debugbar header should be sticky
When you scroll down on a tab it loses the header. That part should stay and the tab contents should be scrollable

### LINKED ISSUE #14: Data does not show when selecting another request
When you select another request, the data does not show until you select another tab

### LINKED ISSUE #15: Make debug provider use the package provider configs

### PR DESCRIPTION
**Pull request by girardinsamuel**
Fixes #15
Fixes #12
Fixes #11 
Fixes #14
- auto load ajax requests in select box

### CODE REVIEW

**Comment by josephmancuso** [comment]
Started adding some changes for service provider to add JSON files to a storage directory

File: src/debugbar/providers/DebugProvider.py [hunk 1]
```diff
@@ -18,13 +23,22 @@ def register(self):
...
+            # Delete the contents of the directory first
+            files = glob.glob('storage/app/debug/*')
+            for f in files:
+                os.remove(f)
+
+            storage.disk('debug').put(f"x{str(time.time())+'-'+random_string(10)}.json", json.dumps(self.application.make('debugger').to_dict()))
             response.content += self.application.make('debugger').get_renderer('javascript').render()
             response.make_headers()
+        else:
+            storage.disk('debug').put(f"x{str(time.time())+'-'+random_string(10)}.json", json.dumps(self.application.make('debugger').to_dict()))
```
Comment (Marlysson) [inline src/debugbar/providers/DebugProvider.py]: @girardinsamuel Isn't this line equal to in 40th line? maybe deletion should be done inside if and others outside the flow?

Comment (girardinsamuel) [reply src/debugbar/providers/DebugProvider.py]: ahah well might be better. for now we are just experimenting with the system. we will try to refactor this cleanly at the end 😉

**Comment by girardinsamuel** [comment]
@josephmancuso A lot of improvements. You can merge that in and test it.
You will need to pull the updates I pushed on the test project so that you can try the ajax requests things.

**Review by josephmancuso — APPROVED** [review]

**Comment by josephmancuso** [comment]
Tested. Works like a charm!

**Pull request merged**